analysis/codelists.py

```diff
@@ -55,3 +55,16 @@
     "codelists/opensafely-pharmacy-first-urinary-tract-infection-treatment-dmd.csv",
     column="code",
 )
+
+pharmacy_first_med_codelist = (
+    acute_otitis_media_tx_cod
+    + impetigo_treatment_tx_cod
+    + infected_insect_bites_tx_cod
+    + shingles_treatment_tx_cod
+    + sinusitis_tx_cod
+    + sore_throat_tx_cod
+    + urinary_tract_infection_tx_cod
+)
+
+pharmacy_first_consultation_codelist = ["1577041000000109", "983341000000102"]
```

Comment:
I think I'd be useful to have comments what each of these codes mean here in this file in case someone is interested in the differences between these codes